Sources/ApolloCodegenLib/Frontend/JavaScript/src/compiler/index.ts

```diff
@@ -28,6 +28,113 @@ import {
 import * as ir from "./ir";
 import { valueFromValueNode } from "./values";
 
+import type { GraphQLOutputType } from 'graphql';
```

Comment:
This was pulled over from the CCN implementation. I'll upstream it once I've got everything else working